state-chain/pallets/cf-vaults/src/lib.rs

```diff
@@ -43,7 +43,12 @@ pub type ThresholdSignatureFor<T, I = ()> =
 #[scale_info(skip_type_params(T, I))]
 pub enum VaultActivationStatus<T: Config<I>, I: 'static = ()> {
 	/// We are waiting for the key to be updated on the contract, and witnessed by the network.
-	AwaitingActivation { new_public_key: AggKeyFor<T, I> },
+	AwaitingActivation {
+		new_public_key: AggKeyFor<T, I>,
+	},
+	ActivationFailedAwaitingGovernance {
+		new_public_key: AggKeyFor<T, I>,
+	},
```

Comment:
@ramizhasan111 this is probably what is causing the CI upgrade test to fail. We should move the new variants below `Complete`.

Reply:
oh yes of course. will update, thanks!